sandbox/src/server.ts

```diff
@@ -29,6 +32,28 @@ const requestLogger: ApolloServerPlugin<MockserverContext> = {
       },
     };
   },
+
+  async unexpectedErrorProcessingRequest({requestContext,error}: {
+    requestContext: GraphQLRequestContext<MockserverContext>;
+    error: Error;
+  }) {
+    log.error({
+        description: 'Unexpected error occurred',
```

Comment:
This error seems GQL specific and of course sandbox specific. The description can be more specific to capture this. Also it should follow our normal logging pattern. https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?spaceKey=RIS&title=Caas+-+Event+Logs

Reply:
I've updated the logging, and added the new values to the page you've linked. Could you have a look and confirm they're correct please?

Reply:
I was expecting to see some error codes in here? Is that I'm not seeing the updates?